.github/workflows/build.yml

```diff
@@ -119,15 +119,20 @@ jobs:
       - get-product-version
       - build
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        arch: ["amd64", "arm64"]
     env:
       repo: ${{github.event.repository.name}}
       version: ${{needs.get-product-version.outputs.product-version}}
 
     steps:
       - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
-      - uses: hashicorp/actions-docker-build@11d43ef520c65f58683d048ce9b47d6617893c9a # v2
+      - uses: hashicorp/actions-docker-build@v2
```

Comment:
Last I checked one of the compliance requirements was that we use exact commits for these, not the tagged version. Has that changed?